modules/custom-css.php

```diff
@@ -13,7 +13,35 @@
  */
 
 function jetpack_load_custom_css() {
-	include dirname( __FILE__ ) . "/custom-css/custom-css.php";
+	// If WordPress has the core version of Custom CSS, load our new version.
+	// @see https://core.trac.wordpress.org/changeset/38829
+	if ( function_exists( 'wp_get_custom_css' ) ) {
+		if ( ! function_exists( 'wp_update_custom_css_post' ) ) {
+			wp_die( 'Please run a SVN up to get the latest version of trunk, or update to at least 4.7 RC1' );
+		}
+		if ( ! Jetpack_Options::get_option( 'custom_css_4.7_migration' ) ) {
+			include_once dirname( __FILE__ ) . '/custom-css/migrate-to-core.php';
+		}
+
+		// TODO: DELETE THIS
```

Comment:
Yeah, not a big deal, and could be helpful if anyone has to revert or re-import.